doc/bind.md

```diff
@@ -28,8 +28,8 @@ const outer = new SimpleMap({
 });
 
 const keyObservable = canValue.bind(outer, "inner.key");
-// canReflect.getValue(keyObservable) === "hello"
+// keyObservable.value === "hello"
 
-canReflect.setValue(keyObservable, "aloha");
+keyObservable.value = "aloha";
```

Comment:
Here’s what this section looks like:
<img width="755" alt="screen shot 2018-07-02 at 7 29 50 pm" src="https://user-images.githubusercontent.com/10070176/42195603-84cf8300-7e2e-11e8-8dee-b8a344b207fb.png">